client/components/fixed-navigation-header/docs/example.jsx

```diff
@@ -1,9 +1,25 @@
+import styled from '@emotion/styled';
 import FixedNavigationHeader from 'calypso/components/fixed-navigation-header';
 
-export default function FixedNavigationHeaderExample() {
+const FixedNavigationHeaderStyled = styled( FixedNavigationHeader )`
```

Comment:
For the purpose of the example, can we remove `styled` usage and inline those styles instead? This seems to generate a warning:

![](https://cldup.com/Ij8Xfpz3Ca.png)

Reply:
Hmm, this is coming from 
https://github.com/Automattic/wp-calypso/blob/269e03dc4fba0c70430440a91d6d5b4c0b4cd62f/client/components/breadcrumb/index.tsx#L19-L23
which I changed to use `:first-of-type` and still works as expected without any warning.